x-pack/plugins/enterprise_search/public/applications/app_search/components/result/result_field_value.scss

```diff
@@ -21,8 +21,3 @@
     font-family: $euiCodeFontFamily;
   }
 }
-
-.enterpriseSearchResultHighlight {
-  color: $euiColorPrimary;
-  font-weight: $euiFontWeightSemiBold;
-}
```

Comment:
@daveyholler - FYI - we're using `euiMark` now instead of our own custom highlighting class. [EuiMark](https://elastic.github.io/eui/#/utilities/highlight-and-mark) doesn't come with color highlighting though - is that an OK change or should we add custom CSS on our result view components to give `.euiMark` a color of `$euiColorPrimary`?

Reply:
Let's stick to vanilla EUI for now and I'll propose the addition of a prop on `EuiMark` for something a little brighter. We're at least seeing a gray background and bold text in the EuiMark right now, correct?

Reply:
There's no gray background, it's just bolded text (same as EUI's docs). I definitely like the option of expanding `EuiMark` itself though, thanks Davey!

Reply:
I'm actually confused why we don't get the background highlight, but it is shown in the demo: https://elastic.github.io/eui/#/utilities/highlight-and-mark

Reply:
There's an additional style that gets applied to `.euiMark` in the demo page, but not within our Kibana page.

Reply:
<img width="898" alt="Elastic_UI_Framework_-_Highlight_and_mark" src="https://user-images.githubusercontent.com/1427475/102650945-a315fa80-4139-11eb-8f55-caec6ee6b810.png">